image/Dockerfile

```diff
@@ -20,10 +20,20 @@ RUN apt-get update && \
     libxrender1 \
     zlib1g-dev \
     libopenexr-dev && \
-    /golem/install_py_libs.sh /golem/work/requirements.txt && \
-    /golem/copy.sh && \
+  /golem/install_py_libs.sh /golem/work/requirements.txt && \
```

Comment:
why did the indent change on these lines?

Reply:
It's not a part of the `apt-get install` command